panels/models/attraction.py

```diff
@@ -1,37 +1,36 @@
+import math
 import pytz
 import re
 
-from collections import OrderedDict
+from collections import OrderedDict, defaultdict
 from datetime import datetime, timedelta
 
 from sideboard.lib import listify
 from sideboard.lib.sa import JSON, CoerceUTF8 as UnicodeText, UTCDateTime, UUID
+from sqlalchemy import and_, exists, func, or_, select, text, union, not_, cast, alias
+from sqlalchemy.ext.associationproxy import association_proxy
```

Comment:
Using an AssociationProxy is one way we might be able to fix [this issue](https://github.com/magfest/ubersystem/issues/3020). See the SQLAlchemy docs on [Simplifying Association Objects](http://docs.sqlalchemy.org/en/latest/orm/extensions/associationproxy.html#simplifying-association-objects). 

By using an AssociationProxy, I believe it forces the association object to be loaded into the session, thus recorded in the tracking table.